src/ctap/storage.rs

```diff
@@ -62,13 +62,11 @@ const PIN_RETRIES: usize = 4;
 const ATTESTATION_PRIVATE_KEY: usize = 5;
 const ATTESTATION_CERTIFICATE: usize = 6;
 const AAGUID: usize = 7;
-#[cfg(not(feature = "with_ctap2_1"))]
-const NUM_TAGS: usize = 8;
 #[cfg(feature = "with_ctap2_1")]
 const MIN_PIN_LENGTH: usize = 8;
 #[cfg(feature = "with_ctap2_1")]
 const MIN_PIN_LENGTH_RP_IDS: usize = 9;
-#[cfg(feature = "with_ctap2_1")]
+// Different NUM_TAGS make the storage incompatible, so we use max(8,10).
```

Comment:
Nit: "Different NUM_TAGS depending on the CTAP version make the storage [...]"

Reply:
I think the current version is better. Because what makes the storage incompatible is different NUM_TAGS. It doesn't matter why we have different NUM_TAGS. In particular, if we add a new tag that is present in all CTAP versions it would still be a breaking change storage-wise.

Note that this issue is only present with v1. The number of keys is fixed in v2.

Reply:
Done, and changed to just say maximum for easier maintenance.